sound/soc/intel/boards/sof_sdw_hdmi.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Copyright(c) 2023 Intel Corporation.
+ */
+
+#ifndef __SOF_SDW_HDMI_H
+#define __SOF_SDW_HDMI_H
```

Comment:
sof_hdmi_common.h and __SOF_HDMI_COMMON_H

Reply:
done